### PR TITLE
Fixes #15060: Make the numbre of LDAP connection of connection pool configuration 

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/15060

### CODE REVIEW

File: webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala [hunk 1]
```diff
@@ -207,6 +207,15 @@ object RudderConfig extends Loggable {
   val LDAP_PORT = config.getInt("ldap.port")
   val LDAP_AUTHDN = config.getString("ldap.authdn")
   val LDAP_AUTHPW = config.getString("ldap.authpw") ; filteredPasswords += "ldap.authpw"
+  val LDAP_MAX_POOL_SIZE = {
+    try {
+      config.getInt("ldap.maxPoolSize")
+    } catch {
+      case ex: ConfigException =>
+        ApplicationLogger.info("Property 'ldap.maxPoolSize' is missing or empty in rudder.configFile. Default to 2 connections.")
+        7
```
Comment (fanf) [inline webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala]: Should be '2'

Comment (ncharles) [reply webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala]: totally

**Comment by ncharles** [comment]
Commit modified

**Comment by fanf** [comment]
OK, merging this PR

**Pull request merged**